### ISSUE TITLE
[ENH] explicit/analytic form of energy function for log-normal distribution

### ISSUE THREAD

**Issue by fkiraly** [sktime/skpro#219]
There does not seem to exist a literature reference for the energy functionals of the log-normal distribution, we should try to drive it, or find a reference.

Collecting discussion below, from https://github.com/sktime/skpro/pull/214.

Current state:
* explicit formula for the cross-term $\mathbb{E}[X-c]$ (almost) derived
* no progress yet on the self-term $\mathbb{E}[X-X']$

@bhavikar04 used Wolfram Alpha to derive the following indefinite integral related to the cross-term $\mathbb{E}[X-c]$:
![Screenshot 2024-03-23 130300](https://github.com/sktime/skpro/assets/77113227/4bacce19-7229-47a3-bf3e-f5a4d88981ae)

My reply:
this looks correct. Now you need to add the limits. That should be an easy substitution, no? I recommend, do that manually. Use that

$\lim_{x\rightarrow -\infty} \mbox{erf}(x) = -1$, and $\lim_{x\rightarrow \infty} \mbox{erf}(x) = 1$. You need to be careful with the sign, but that should be it?

The number 0.707 etc should be $\frac{1}{2} \sqrt{2}$, but it doesn't matter for the limits.

**Comment by bhavikar04** [comment]
For the cross-Energy, when using limits x=0--->inf, it seems to evaluate to 0 except when c=0.

**Comment by fkiraly** [comment]
> For the cross-Energy, when using limits x=0--->inf, it seems to evaluate to 0 except when c=0.

That cannot be, you must be making an error of sign or similar...

There exists no distribution for which the cross-term is almost always 0.

**Comment by bhavikar04** [comment]
> > For the cross-Energy, when using limits x=0--->inf, it seems to evaluate to 0 except when c=0.
> 
> That cannot be, you must be making an error of sign or similar...
> 
> There exists no distribution for which the cross-term is almost always 0.

same thoughts but rechecking gives me the same answer, to confirm: as x-->inf erf functions in the expression become -1 because of the (-logx) term and similarly become 1 when x-->0, right?

**Comment by fkiraly** [comment]
yes, but the `sgn` at the start also flips, cancelling these out. So you get a factor of 2, not a factor of 0.

**Comment by fkiraly** [comment]
There might be sth going on with the wolfram heuristic being buggy. What is that strange "plus zero" in the sign function?
